codeforamerica/vita-min
# [FYST-1262] More Idaho Personas

## Link to pivotal/JIRA issue
[FYST-1262](https://codeforamerica.atlassian.net/browse/FYST-1262)
## Is PM acceptance required? (delete one)
Yes? - don't merge until JIRA issue is accepted

## What was done?
Added two Idahoans to our DF personas per request from Erica.
## How to test?
Ensure you can successfully import `Klein_qss` and `Estrada_donations` in Idaho.


[FYST-1262]: https://codeforamerica.atlassian.net/browse/FYST-1262?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
